share/module_bc.F

```diff
@@ -2615,6 +2615,113 @@ SUBROUTINE flow_dep_bdy_qnn  (  field,          &
 
    END SUBROUTINE flow_dep_bdy_qnn
 
+!-------------------------for ntu3m-------------------------------------------------
+   SUBROUTINE flow_dep_bdy_ntu(field,u,v,config_flags,spec_zone,ids,   &
+                               ide,jds,jde,kds,kde,ims,ime,jms,jme,kms,&
+                               kme,ips,ipe,jps,jpe,kps,kpe,its,ite,jts,&
+                               jte,kts,kte)
+!-------------------------------------------------------------------------------
```

Comment:
I would like to know why this new boundary condition is needed and especially why changes are needed for periodic conditions. The Registry has the PERIOD halos that may address these.

Reply:
The background aerosols (QDCN, QTCN, and QINI) are needed for the initiation of liquid droplets (activation) and ice particles (nucleation) in the NTU scheme. If we do not specify aerosol boundary conditions, this may cause these inertial aerosols to vanish as a result of advection after a long simulation time. The below figure is one example of a real-case simulation with the default boundary condition (left panels) of QDCN at different time periods. Currently, the zero value (in flow_dep_bdy) and constant of ccn_conc (in flow_dep_bdy_qnn) assigned for inflow in the boundary specified region are not suitable for the vertical damping of aerosols in the NTU scheme. The right panels depict the revised results with the modified flow_dep_bdy_ntu. It will be appreciated if you have any suggestions. 

![image](https://user-images.githubusercontent.com/73516426/108478986-21c11c80-72d0-11eb-9e91-ebd0767e3b48.png)